docs/resources/db_target.md

```diff
@@ -0,0 +1,263 @@
+
+Create a Db target with the default authentication configuration
+(non-passwordless) and a proxy environment. When a user connects to this target,
+the Bzero or Cluster target with the least number of open connections in this
+environment is used to proxy the connection to the configured database.
+
+```terraform
+data "bastionzero_environments" "example" {}
+
+locals {
+  # Find environment with name "example-env". `env` is null if not found
+  env = one([
+    for each in data.bastionzero_environments.example.environments
+    : each if each.name == "example-env"
+  ])
+}
+
+resource "bastionzero_db_target" "example" {
+  name                 = "example-psql-db-target"
+  remote_host          = "localhost"
+  environment_id       = local.env.id
+  remote_port          = 5432
+  proxy_environment_id = local.env.id
+  # Configures the Db daemon to run on port 5432 when a user connects to this Db
+  # target
+  local_port = 5432
+}
+```
+
+### SplitCert Db target
+
+Create a Db target with SplitCert enabled; SplitCert provides passwordless
+access to databases. In this example, the Db target is configured as a
+self-hosted Postgres database. Learn more
+[here](https://docs.bastionzero.com/docs/how-to-guides/passwordless-database-access/passwordless-access-to-self-hosted-postgres).
+
+```terraform
+data "bastionzero_environments" "example" {}
+data "bastionzero_bzero_targets" "example" {}
+
+locals {
+  # Find environment with name "example-env". `env` is null if not found
+  env = one([
+    for each in data.bastionzero_environments.example.environments
+    : each if each.name == "example-env"
+  ])
+  # Find Bzero target with name "ubuntu". `proxy_target` is null if not found
+  proxy_target = one([
+    for each in data.bastionzero_bzero_targets.example.targets
+    : each if each.name == "ubuntu"
+  ])
+}
+
+resource "bastionzero_db_target" "example" {
+  name            = "example-splitcert-db-target"
+  remote_host     = "localhost"
+  environment_id  = local.env.id
+  remote_port     = 5432
+  proxy_target_id = local.proxy_target.id
+  database_authentication_config = {
+    authentication_type = "SplitCert"
+    database            = "Postgres"
+    label               = "Unmanaged Postgres"
+  }
+}
+```
+
+### Passwordless access to Postgres on GCP Cloud
+
+Create a Db target using service accounts which provides passwordless access to
+the database. Learn more
+[here](https://docs.bastionzero.com/docs/how-to-guides/passwordless-database-access/passwordless-access-to-mysql-and-postgres-on-gcp-cloud-sql).
+
+```terraform
+data "bastionzero_environments" "example" {}
+data "bastionzero_bzero_targets" "example" {}
+
+locals {
+  # Find environment with name "example-env". `env` is null if not found
+  env = one([
+    for each in data.bastionzero_environments.example.environments
+    : each if each.name == "example-env"
+  ])
+  # Find Bzero target with name "ubuntu". `proxy_target` is null if not found
```

Comment:
Find Linux target

Reply:
This data source list can include Linux or Windows targets, so I've changed it to that. Lmk if that's okay.